global_planner/launch/global_planner_sitl_3cam.launch

```diff
@@ -0,0 +1,39 @@
+<launch>
+
+    <arg name="world_file_name"    default="simple_obstacle" />
+    <arg name="world_path" default="$(find avoidance)/sim/worlds/$(arg world_file_name).world" />
+    <arg name="pointcloud_topics" default="[/camera_front/depth/points,/camera_left/depth/points,/camera_right/depth/points]"/>
+    <arg name="start_pos_x" default="0.5" />
+    <arg name="start_pos_y" default="0.5" />
+    <arg name="start_pos_z" default="3.5" />
+
+    <!-- Define a static transform from a camera internal frame to the fcu for every camera used -->
+    <node pkg="tf" type="static_transform_publisher" name="tf_local_origin"
+          args="0 0 0 0 0 0 world local_origin 10"/>
+    <node pkg="tf" type="static_transform_publisher" name="tf_camera"
```

Comment:
is this static transform necessary?

Reply:
I am having a hard time understanding why, but deleting it results breaking the system

Reply:
I think we can delete them